docs/README.md

```diff
@@ -0,0 +1,27 @@
+# tkn
+`tkn` is the Tekton pipeline command line interface (CLI)
+
+## Getting Started
+
+### Installing `tkn`
+
+You can grab the latest nightly binary executalbe for :
+# Note:- Need to update below links.
+* [Max OS X](The/URL/to/binary)
+* [Linux AMD 64](The/URL/to/binary)
+* [Windows AMD 64](The/URL/to/binary)
+
+## Need to add installation steps.
+
```

Comment:
Should there be a section with the list of commands, linked to their respective doc page?

Reply:
Adding all the commands will make it large which will definitely grow soon.
How about adding links to all the parent commands docs like pipeline, pipelinerun, task, taskrun which further have links to their child commands?